src/Ztm.WebApi/InvalidCallbackUrlException.cs

```diff
@@ -1,24 +1,20 @@
+using System;
 using System.Net;
 
 namespace Ztm.WebApi
 {
     public sealed class InvalidCallbackUrlException : HttpResponseException
     {
-        public InvalidCallbackUrlException(string InvalidUrl)
+        static readonly string InvalidCallbackUrlTitle = "Invalid Callback Url";
```

Comment:
We should not apply naming convention on display text. My idea is change it value to `Invalid callback URL.`.